### ISSUE TITLE
Add documentation to help

### ISSUE THREAD

**Issue by alexarje** [fourMs/VideoAnalysis#16]
I found the documentation under About, but it is possible to also get it as an entry under Help in the menu?

**Comment by balintlaczko** [comment]
Yes, that was the way I envisioned it originally, but then while working with the `[menubar]` object, it became clear that it only allows an about section, and you cannot put anything in the help - at least I haven't found an obvious way yet. That is way it ended up as a _half-easter-egg_ in the About section (since in Windows, About is also under Help). I will try it again now! In worst case we can always just make another menu item for it.

**Comment by balintlaczko** [comment]
Indeed can re-confirm, that it is not allowed to put anything in the Help menu. 
However, this should not stop us doing it anyway. After some [research](https://cycling74-web-uploads.s3.amazonaws.com/54d0d34d6bd55921179c8caf/2017-06-21T14:20:09Z/expo74-DanNigrin_standalones.pdf) and digging, I could manage to remove all the useless _things_ from the menu + adding my new Help menu item by editing the _./resources/interfaces/maxinterface.json_ . Here is how it looks like now:
![image](https://user-images.githubusercontent.com/45127463/75498057-6b7d6c80-59c6-11ea-93bf-cebffb437f0d.png)

In File:
![image](https://user-images.githubusercontent.com/45127463/75498085-80f29680-59c6-11ea-9263-b7f0803cfcbd.png)

...and in Help:
![image](https://user-images.githubusercontent.com/45127463/75498100-949dfd00-59c6-11ea-8229-6a8e8026224a.png)

There is my new clickable Help entry. I have to test this though, it is hard to see at this point if the clicking can be listened to. If the listening works though, this means we finally have a full control over how our menu looks like. The changes seemingly kept already existing functionality and behavior intact.

**Comment by balintlaczko** [comment]
After some more tweaking of the _maxinterface.json_ I have this layout now:
![image](https://user-images.githubusercontent.com/45127463/75553699-2f372400-5a39-11ea-99d1-04d4a7e31327.png)
In File:
![image](https://user-images.githubusercontent.com/45127463/75553750-4b3ac580-5a39-11ea-966c-e940934b5c55.png)
In Help:
![image](https://user-images.githubusercontent.com/45127463/75553767-55f55a80-5a39-11ea-85e8-31c8c6a0f451.png)
I will start to test now if we can listen to click events from these new custom entries.

**Comment by balintlaczko** [comment]
Another status report after a bit more "brain surgery" of Max: though it _is_ possible to create custom menu items by manually modifying the _maxinterface.json_ there is a problem with this approach, namely that the interpreter/compiler of the `[menubar]` script still won't let you use them, and I found no way to bypass this. What is simpler, and maybe even more consistent is to purge the "normal" File and Help menus all together from _maxinterface.json_, and create aliases in the `[menubar]` script. This way I got the same-looking result as in the previous comment (without the keyboard shortcuts) but with being able to listen to my File and Help menu items the same way as any other menu items in the `[menubar]`.
Bottom line is that through some mild hacking we will be able to:
- get rid of the usual "bloat" menus and menu items (which tend to confuse new users in my experience)
- have separate _Open File_ and _Open folder_ menu events as we always wanted
- have _Get Started..._ in the "root" Help menu (this issue)
- have the _Report a problem..._ in the "root" Help menu (#14 )
- and ultimately a technique which gives complete control of the menu design (could be reused in future max standalones too).

I will start to implement the necessary changes in the root patch and `[VA_menubar]`, and get back here with the results.

**Comment by balintlaczko** [comment]
Done! NB. if your _maxinterface.json_ is the default one you won't be able to see this change in the patch, only in the build (and only if you follow the instructions in _build_info/build_checklist.txt_). Alternatively, you can do what I did on my system: swapping the _maxinterface.json_ found at (on Windows) _C:\\Program Files\Cycling'74\Max 8\resources\interfaces\maxinterface.json_. But back up the original one, just in case. That way it will work in the patch too.
Here is how it looks in the build:
![image](https://user-images.githubusercontent.com/45127463/75567920-9e6d4200-5a52-11ea-9372-09cc22d211b6.png)
Upon clicking on this, the embedded doc browser opens. Since we moved it from the About window, that one looks like this now:
![image](https://user-images.githubusercontent.com/45127463/75568032-d2486780-5a52-11ea-9566-f22ce9128633.png)

**Comment by alexarje** [comment]
Great, well done!
